### PR TITLE
Add Podcast Promo Link

### LINKED ISSUE #5427: Podcast Promo Link
**Is your feature request related to a problem? Please describe.**
The promo link has a podcast option so we can indicate the link is to the page containing a podcast. The difference between this and a standard link is that for a standard link we show the time/date of when the promo was created which is good to show for the 'freshness' of an article but bad if we want to use it to link to the page containing the podcast.
https://jira.dev.bbc.co.uk/browse/WSFRONTPGS-281

**Describe the solution you'd like**

- Add `Podcast` to the contentType and display podcast promo if the asset type is PRO. This `Podcast` promo should not have the timestamp displayed. `"contentType": "Podcast", "assetTypeCode": "PRO"`

- Add podcast promo examples to the frontpage.json so that we can test it, `Nepali, Indonesia, Gahuza, Burmese` services that have podcasts, we should add examples for these

- Update unit tests.

- Remove the date from the promo for podcast promos

**Describe alternatives you've considered**

**Testing notes**
The podcast promo should on frontopage should link to the page containing the podcast.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
example of whitelist video content: 
https://github.com/bbc/simorgh/issues/3540 
https://github.com/bbc/simorgh/pull/3601

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #5427

**Overall change:** 
Added a podcast promo link to whitelist and made podcasts render without a date/timestamp.

Screenshot 2020-02-10 at 15 11 13

**Code changes:**
- Added "Podcast" content type to whitelist
- Added a condition so that podcasts are displayed without a date
- Added tests for filtering podcasts and snapshots for displaying podcasts without duration
- Added podcasts to `StoryPromo` storybook
- Added podcast data for Pidgin since it is required to display a Podcast story in Storybook

Nepali, Indonesia, Gahuza and Burmese services already had examples of Podcast data in their `frontpage.json` files so did not require updates.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -3426,6 +3426,442 @@ exports[`StoryPromo Container should render multiple Index Alsos correctly for c
...
...QuODQ3LTEzLjUyNi03Ljk5Ny0yNC42NTQtNy45OTFIMzcuODN2OTcuNzI4aDM2LjA3M2MxMi44NyAwIDIxLjkwNi0zLjQ4MiAyNy43MjItOC42NSA1LjgxOC01LjE1NSA4LjMyLTExLjkxIDguMzEyLTE4LjEzOHpNMCAxMzUuNjk4aDE0NC4xOTdWMEgwdjEzNS42OTd6Ii8+PHBhdGggZD0iTTI1OC42NjIgODguMTk4Yy0uMDEzIDMuMjI5LTEuMDA3IDYuNDc1LTMuODk2IDkuMDExLTIuODg0IDIuNTM3LTcuODczIDQuNDYzLTE2LjEzMyA0LjQ2M0gyMjJWNzVoMTUuODkzYzcuNDExIDAgMTIuNjcgMS41MDIgMTUuOTY1IDMuODUgMy4yODkgMi4zNjIgNC43NzYgNS40NjMgNC44MDQgOS4zNDgiLz48cGF0aCBkPSJNOTEuNjYyIDg4LjE5OGMtLjAxMyAzLjIyOS0xLjAwNyA2LjQ3NS0zLjg5NiA5LjAxMS0yLjg4NCAyLjUzNy03Ljg3NCA0LjQ2My0xNi4xMzMgNC40NjNINTVWNzVoMTUuODkyYzcuNDEyIDAgMTIuNjcyIDEuNTAyIDE1Ljk2NiAzLjg1IDMuMjg5IDIuMzYyIDQuNzc2IDUuNDYzIDQuODA0IDkuMzQ4Ii8+PHBhdGggZD0iTTI0NS4xODYgNTUuNzljMy4wOTYtMi4yMzcgNC41OS01LjM4NiA0LjYxMy0xMC4xMjQtLjAxNS0zLjI1LS45NDMtNi4wMzMtMy4yODEtOC4xMTEtMi4zNDYtMi4wNzgtNi4zMy0zLjU1NS0xMi43NTQtMy41NTVIMjIydjI1LjI3NWg4LjA3NmM2Ljk4OC4wMDQgMTEuOTk4LTEuMjQzIDE1LjExLTMuNDg2Ii8+PHBhdGggZD0iTTc4LjE4NiA1NS43OWMzLjA5Ni0yLjIzNyA0LjU5LTUuMzg2IDQuNjEzLTEwLjEyNC0uMDE1LTMuMjUtLjk0My02LjAzMy0zLjI4Mi04LjExMUM3Ny4xNzIgMzUuNDc3IDczLjE4OCAzNCA2Ni43NjQgMzRINTV2MjUuMjc1aDguMDc2YzYuOTg4LjAwNCAxMS45OTgtMS4yNDMgMTUuMTEtMy40ODYiLz48L2c+PC9nPjwvZz48L3N2Zz4K);
```
Comment (greenc05) [inline src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap:3731]: Just wondering what this image is? BBC blocks?

File: src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -3426,6 +3426,442 @@ exports[`StoryPromo Container should render multiple Index Alsos correctly for c
...
+    width: initial;
+    padding: initial;
+    grid-column: 3 / span 4;
+  }
+}
+
+<div
+  class="c0"
+>
+  <div
```
Comment (greenc05) [inline src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap:3829]: what are all these divs for??

File: src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (OlgaLyubin) [reply src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap:3731]: Yes, those are BBC blocks

File: src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap (hunk 2 above)
Comment (OlgaLyubin) [reply src/app/containers/StoryPromo/__snapshots__/index.test.jsx.snap:3829]: `<div class="c0">` - For outer Grid of Story Promo
`<div class="c1">` - Part of the Grid that takes up 1/3 of the outer grid and displays an Image
`<div class="c2">` - Image Contents Wrapper
`<div class="c3">` - Image Placeholder (the one that contains an actual image inside)
`<div class="c4">` - Inline Media Indicator (this gets rendered in DOM even when the Story Promo doesn't display a media indicator, that's just the way it works in Simorgh now)
`<div class="c5">` - Part of the Grid that takes up the remainder 2/3 of the outer grid and displays a Headline+Summary

**Review by mmcculloch — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
Great work on this 👍

**Comment by OlgaLyubin** [comment]
This PR has been approved by UX

**Comment by PriyaKR** [comment]
Looks good to me 

http://localhost:7080/pidgin
http://localhost:7080/pidgin.amp
http://localhost:7080/burmese
http://localhost:7080/burmese.amp

**Pull request merged**